src/Spice86.Core/Emulator/Function/FunctionHandler.cs

```diff
@@ -209,9 +202,6 @@ public void Ret(CallType returnCallType, IReturnInstruction? ret) {
 
     private bool HandleReturn(CallType returnCallType, FunctionCall currentFunctionCall) {
         bool returnAddressAlignedWithCallStack = DetectUnalignedReturns(currentFunctionCall, returnCallType);
-        if (_executionFlowRecorder == null) {
-            return returnAddressAlignedWithCallStack;
-        }
         FunctionInformation? currentFunctionInformation = _functionCatalogue.GetFunctionInformation(currentFunctionCall);
```

Comment:
In cfgcpu, _executionFlowRecorder is null (re-instanciated function handler managed by cfgcpu).
This means the code below needs to be removed as well.
AddReturn, AddUnalignedReturn and IsReturnRecorded of FunctionInformation should be removed.